epictrack-web/src/components/shared/controlledInputComponents/ControlledSelectV2.tsx

```diff
@@ -13,6 +13,7 @@ type IFormInputProps = {
   helperText?: string | undefined;
   disabled?: boolean;
   onHandleChange?: (val: any) => void;
+  testId?: string;
```

Comment:
this can be part of otherProps and I don't think we have to explicitly define it